data_validation/config_manager.py

```diff
@@ -508,6 +508,55 @@ def build_config_manager(
             verbose=verbose,
         )
 
+    def add_rstrip_to_comp_fields(self, comparison_fields: List[str]) -> List[str]:
+        """As per #1190, add an rstrip calculated field for Teradata string comparison fields.
+
+        Parameters:
```

Comment:
For my education, do you use a VS Code extension to auto create docstrings or type them out yourself?

Reply:
Type them out myself :)